pysilcam/silcamgui/interactive_summary.py

```diff
@@ -229,8 +230,9 @@ def __init__(self, parent=None):
         self.axisconstant = plt.subplot(221)
         self.axispsd = plt.subplot(122)
         self.axistext = plt.subplot(223)
-        plt.sca(self.axistext)
-        plt.axis('off')
+        #plt.sca(self.axistext)
+        #plt.axis('off')
```

Comment:
E265: block comment should start with '# '  <!-- Automatically posted by Lintly -->